### ISSUE TITLE
T-Bar control

### ISSUE THREAD

**Issue by pmaillot** [jshea2/OSC-for-OBS#22]
Any chance to have an OSC command set to control the OBS T-BAR? The latest OBS doc mentions the following:
Set the Transition Bar position (in Studio Mode). If you need to perform multiple successive T-Bar moves (e.g. : in an animation, or in response to a user moving a T-Bar control in your User Interface), set release to false and execute Release Transition Bar command later once the animation/interaction is over.

Name    | Type      | Description
Position | Float     | Value must be between 0.0 and 1.0.
Release | Boolean | True or False

**Comment by jshea2** [comment]
Yeah, I can definitely look into adding it. I'm unfamiliar with this feature, how would the syntax for the OSC command make the most sense to you?

**Comment by pmaillot** [comment]
This would be fantastic!

A form like
/tbar ,fi [0..1] [0/1] would be great
with /tbar being the OSC address, taking two OSC parameters, the first one being float 0 to 1 and the second one being int 0 or 1.

I'd guess a good use case would be send a successive set of commands
/tbar ,fi <tbar_value><0>
and on the last user move of the T-bar (reaching max for example, or after a timeout)
/tbar ,fi <tbar_value><1>

The [1] value for release would snap the T-Bar to its initial position, not changing the resulting transition that took place, and preparing for the next to come transition.

On automated transitions (such as fade, with a transition time of 3000 for ex) one can imagine the program to run the equivalent of /tbar ,fi <value> <0>, with value from 0.0 to 1.0, fading to the new scene in 3000ms, and releasing the T-Bar to its initial 0 position once done.
